tests/test_organizations/test_organizations_boto3.py

```diff
@@ -410,28 +410,72 @@ def test_describe_policy_exception():
 
 
 @mock_organizations
-def test_attach_policy():
```

Comment:
Don't think this test needs to change? The attach_policy functionality hasn't changed